ui/brave_custom_notification/ad_notification_view_md.h

```diff
@@ -0,0 +1,133 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_UI_BRAVE_CUSTOM_NOTIFICATION_AD_NOTIFICATION_VIEW_MD_H_
+#define BRAVE_UI_BRAVE_CUSTOM_NOTIFICATION_AD_NOTIFICATION_VIEW_MD_H_
+
+#include <memory>
+#include <vector>
+
+#include "base/gtest_prod_util.h"
+#include "base/macros.h"
+#include "base/optional.h"
+#include "base/time/time.h"
+#include "brave/ui/brave_custom_notification/notification_view.h"
+#include "ui/views/animation/ink_drop_observer.h"
+#include "ui/views/controls/button/button.h"
+#include "ui/views/controls/button/label_button.h"
+#include "ui/views/controls/textfield/textfield_controller.h"
+
+namespace views {
+class ImageButton;
+class Label;
+class LabelButton;
+class ProgressBar;
+class RadioButton;
+class Textfield;
```

Comment:
Can we please remove these unsupported types and any related code, as ad notifications do not support these views

Reply:
Can we please remove any code which is not used for brave ads